pkg/networkmanager/network-main.jsx

```diff
@@ -138,6 +138,46 @@
 
     const actions = privileged && (
         <>
+            <Button onClick={ async () => {
+                const key = await cockpit.script("wg genkey");
+
+                const client = cockpit.dbus("org.freedesktop.NetworkManager", { superuser: 'try' });
+                try {
+                    const reply = await client.call("/org/freedesktop/NetworkManager/Settings",
+                                                    "org.freedesktop.NetworkManager.Settings",
+                                                    "AddConnection",
```

Comment:
These 5 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19024-20230626-161008-14729dcc-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)